src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -318,13 +323,20 @@
     default-so-linger-in-milliseconds))
 
 (schema/defn ^:always-validate
+  common-connector-config :- WebserverConnectorCommon
```

Comment:
Good call on factoring this out.  The settings repetition between the http and https connectors was starting to get ugly.